portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,15 +35,62 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private List<String> names;
+
+  private List<String> messages;
+
   @Override
   public void init(){
     names = new ArrayList<>();
     names.add("Obi");
+
+    messages = new ArrayList<>();
+    messages.add("Hello! How are you");
+    //messages.add("You got this!");
```

Comment:
Typically we drop commented out code in industry to keep things as clean & short as possible.